rfcs/IncrementalDelivery.md

```diff
@@ -6,9 +6,15 @@ This RFC proposes adding a specification of how "Incremental" GraphQL results sh
 
 An Incremental result is a GraphQL result that is split up into multiple payloads, allowing clients quicker access to parts of the results. Currently this is supported by the proposed `@defer` and `@stream` directives ([RFC](https://github.com/graphql/graphql-spec/blob/master/rfcs/DeferStream.md)).
 
-## `transfer-encoding: chunked`
+### HTTP/1.1
 
-The HTTP response for an incrementally delivered response should contain the `transfer-encoding: chunked` response header. Chunked transfer encoding allows the body of the response to be delivered as a series of chunks, allowing clients to read each chunk of the response as it is sent without waiting for the entire response.
+An incrementally delivered response should contain the `transfer-encoding: chunked` response header when using HTTP/1.1. Chunked transfer encoding allows the body of the response to be delivered as a series of chunks, allowing clients to read each chunk of the response as it is sent without waiting for the entire response.
```

Comment:
I think we should always capitalize HTTP headers `Like-This: value` in this entire document.

Reply:
I was matching the style to keep the changeset at the minimum - I can follow-up with a PR to take care of styling issues.